### PR TITLE
Now, Vinegar has a GUI.

### PR DESCRIPTION
**Pull request by BlueTree242**

### CODE REVIEW

**Comment by Cristiandis** [comment]
No, its just a loading screen
Gui in terms of config there's none

**Comment by Noted-Jrelvas** [comment]
According to @BlueTree242's patch, this is a Graphical User Interface.

imagem

**Comment by Cristiandis** [comment]
> According to @BlueTree242's patch, this is a Graphical User Interface.
> 
> imagem

Real

**Comment by Cristiandis** [comment]
Close this pr

**Comment by apprehensions** [comment]
No

File: src/Home/index.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 
 - Vinegar is a transparent, minimal, configurable, and fast bootstrapper for Roblox Player and Roblox Studio.
 - The software and documentation are both open source, and can be accessed [on GitHub](https://github.com/vinegarhq).
-- There is no <abbr title="Graphical User Interface">GUI</abbr> for Vinegar at this time.
+- There is no <abbr title="Graphical User Interface">GUI</abbr> for Vinegar at this time, but there's a loading screen.
```
Comment (tacheometry) [inline src/Home/index.md:9]:
```suggestion
- There is no graphical configuration interface for Vinegar at this time.
```

**Review by tacheometry — CHANGES_REQUESTED** [review]

**Review by tacheometry — APPROVED** [review]

**Pull request merged**